docs/dht-best-ping.md

```
@@ -0,0 +1,114 @@
+            Value:  Peer{Addr: "peer5.example:4001", Ping: 30},
+        }
+        _ = kdt.Insert(improved)
+    }
+
+    // Recompute the best after update
+    best2, d2, _ := kdt.Nearest([]float64{0})
+    fmt.Printf("After update, best peer: %s (ping=%d ms), distance=%.0f\n", best2.Value.Addr, best2.Value.Ping, d2)
+}
+```
+
+### Why does querying with `[0]` work?
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)